### DISCUSSION TITLE
help me to configure Qualcomm Adreno 506 gpu

### DISCUSSION THREAD

**Discussion by srirengarajan-pb-16331** [blakeblackshear/frigate#13882] [General]
hi team,

I am frigate in redmi note 4, via post market OS using docker
it has Qualcomm Adreno 506 gpu

i have found these things
xiaomi-mido:~$ v4l2-ctl --device=/dev/video6 --get-fmt-video
Format Video Capture Multiplanar:
	Width/Height      : 1280/736
	Pixel Format      : 'H264' (H.264)
	Field             : None
	Number of planes  : 1
	Flags             :
	Colorspace        : Default
	Transfer Function : Default
	YCbCr/HSV Encoding: Default
	Quantization      : Default
	Plane 0           :
	   Bytes per Line : 0
	   Size Image     : 708608
	   
xiaomi-mido:~$ ffmpeg -hwaccels
ffmpeg version 6.1.1 Copyright (c) 2000-2023 the FFmpeg developers
  built with gcc 13.2.1 (Alpine 13.2.1_git20240309) 20240309
  configuration: --prefix=/usr --disable-librtmp --disable-lzma --disable-static --disable-stripping --enable-avfilter --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libdrm --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libharfbuzz --enable-libmp3lame --enable-libopenmpt --enable-libopus --enable-libplacebo --enable-libpulse --enable-librav1e --enable-librist --enable-libsoxr --enable-libsrt --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-libzmq --enable-lto=auto --enable-lv2 --enable-openssl --enable-pic --enable-postproc --enable-pthreads --enable-shared --enable-vaapi --enable-vdpau --enable-version3 --enable-vulkan --optflags=-O3 --enable-libjxl --enable-libsvtav1
  libavutil      58. 29.100 / 58. 29.100
  libavcodec     60. 31.102 / 60. 31.102
  libavformat    60. 16.100 / 60. 16.100
  libavdevice    60.  3.100 / 60.  3.100
  libavfilter     9. 12.100 /  9. 12.100
  libswscale      7.  5.100 /  7.  5.100
  libswresample   4. 12.100 /  4. 12.100
  libpostproc    57.  3.100 / 57.  3.100
Hardware acceleration methods:
vdpau
vaapi
drm
vulkan

i have passed these device to docker 
docker run -d \
  --name frigate \
  --restart=unless-stopped \
  --mount type=tmpfs,target=/tmp/cache,tmpfs-size=1000000000 \
  --shm-size=64m \
  --device /dev/dri/renderD128:/dev/dri/renderD128 \
  --device /dev/video6:/dev/video6 \
  -v /home/renga/frigate_storage:/media/frigate \
  -v /home/renga/frigate_config:/config \
  -v /etc/localtime:/etc/localtime:ro \
  -e FRIGATE_RTSP_PASSWORD='password' \
  -p 8971:8971 \
  -p 8554:8554 \
  -p 8555:8555/tcp \
  -p 8555:8555/udp \
  -p 5000:5000 \
  ghcr.io/blakeblackshear/frigate:stable
  
  in frigate log 
  CPU detectors are not recommended and should only be used for testing or for trial purposes
  
  please guide me how to configure it to run on GPU

**Comment by hawkeye217** [comment]
> `CPU detectors are not recommended and should only be used for testing or for trial purposes`

This is a log message related to Frigate's [object detection](https://docs.frigate.video/configuration/object_detectors/) and doesn't have anything to do with decoding your video streams.

You'll need to manually specify ffmpeg arguments if you want to use non-standard [hardware accelerated stream decoding](https://docs.frigate.video/configuration/hardware_acceleration).

**Reply by srirengarajan-pb-16331** [reply]
thanks
